starter-code/src/main.js

```diff
@@ -40,8 +120,64 @@ $(document).ready(function(){
 
   // Bind the click event of each element to a function
   $('.back').click(function () {
-    // TODO: write some code here
+    console.log(clicked); // for my revision
+
+    $(this).hide(); // hide div wit back function 
+    $(this).next().toggleClass("back"); // but also give .back class to div next to .back. So it has now background image + width and height
```

Comment:
You used several lines of code to toggle the classed of the div's inside the card div. This is not incorrect, but you could have use less code for example with the following line: 

$(this).children().toggleClass('front back');

This same line can be used to toggle back both cards that are in the picked array, check my comment on line 157 :)